packages/hardhat/contracts/strategy/CrabStrategyV2.sol

```diff
@@ -181,11 +188,44 @@ contract CrabStrategyV2 is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Own
         require(msg.sender == weth || msg.sender == address(powerTokenController), "Cannot receive eth");
     }
 
+    /**
+     * @notice initializes the collateral ratio upon the first migration
+     */
+     function initialize(uint256 wSqueethToMint) external payable { 
```

Comment:
note to myself: Remember to only allow CrabMigration to call this if not big problems.